configs/ext/tfm_mbedcrypto_config_profile_medium.h

```diff
@@ -325,6 +325,28 @@
  */
 #define MBEDTLS_AES_C
 
+/**
+ * \def MBEDTLS_BLOCK_CIPHER_NO_DECRYPT
+ *
+ * Remove decryption operation for AES, ARIA and Camellia block cipher.
+ *
+ * \note  This feature is incompatible with insecure block cipher,
+ *        MBEDTLS_DES_C, and cipher modes which always require decryption
+ *        operation, MBEDTLS_CIPHER_MODE_CBC, MBEDTLS_CIPHER_MODE_XTS and
+ *        MBEDTLS_NIST_KW_C. When #MBEDTLS_PSA_CRYPTO_CONFIG is enabled,
+ *        this feature is incompatible with following supported PSA equivalence,
+ *        PSA_WANT_ALG_ECB_NO_PADDING, PSA_WANT_ALG_CBC_NO_PADDING,
+ *        PSA_WANT_ALG_CBC_PKCS7 and PSA_WANT_KEY_TYPE_DES.
+ *
+ * Module:  library/aes.c
+ *          library/aesce.c
+ *          library/aesni.c
+ *          library/aria.c
+ *          library/camellia.c
+ *          library/cipher.c
+ */
+#define MBEDTLS_BLOCK_CIPHER_NO_DECRYPT
```

Comment:
I think we should also remove/comment out `MBEDTLS_AES_SETKEY_DEC_ALT` and `MBEDTLS_AES_DECRYPT_ALT` in this file: it was a hack that was necessary because we didn't have a proper option for that, but now that we do it's no longer useful. (Also, remove `#undef`ing of those symbols in `configs/config-tfm.h` then.)

Edit: I see now that's done in #8535, so everything's fine actually.